### PR TITLE
Test Centrestack Installation Documentation Update

### PR DESCRIPTION
**Pull request by BryanGuo302**

### CODE REVIEW

File: install/source/chapter02.rst [hunk 1]
```diff
@@ -134,18 +134,18 @@ CentreStack will install and then you will need to restart your system to comple
...
 
-.. image:: _static/image014.png
+.. image:: _static/image_s2_5_5_v3.png
 
 And Click Install.
 
-.. image:: _static/image015.png
+.. image:: _static/image_s2_5_6_v2.png
 
 Once installation is complete, a reboot is required. Click 'Yes' on the reboot window.
```
Comment (anabelpgladinet) [inline install/source/chapter02.rst:151]: The new screenshot is different, change it for Restart ....

File: install/source/chapter02.rst [hunk 2]
```diff
@@ -134,18 +134,18 @@ CentreStack will install and then you will need to restart your system to comple
 
 For the next screen, if you have an existing database use the pull-down menu to select a different option otherwise use the default which will also install the MySQL Community Edition database.
 
-.. image:: _static/image_s2_5_4.png
+.. image:: _static/image_s2_5_7_v2.png
 
 In the next screen, enter your Partner Account login credentials to complete the installation.
 
-.. image:: _static/image_s2_5_5.png
+.. image:: _static/image_s2_5_8_v2.png
```
Comment (anabelpgladinet) [inline install/source/chapter02.rst]: Blur the email

File: install/source/chapter02.rst [hunk 3]
```diff
@@ -112,19 +112,19 @@ Step 5: Start Installation
 
 You can get the CentreStack installation package from the CentreStack partner portal (by login to http://www.centrestack.com) and get to the Private CentreStack section.
 
-.. image:: _static/image_s2_5_1.png
+.. image:: _static/image_s2_5_1_v2.png
```
Comment (anabelpgladinet) [inline install/source/chapter02.rst]: Check if arrow is better with greenshot tool

File: install/source/chapter02.rst (hunk 1 above)
Comment (anabelpgladinet) [inline install/source/chapter02.rst]: The new screenshot is different, now could be something like:
Once installation is complete, a reboot is required. Click 'Restart' on the Installation Completed window.

File: install/source/chapter02.rst [hunk 4]
```diff
@@ -112,19 +112,19 @@ Step 5: Start Installation
...
 
 Sign into the CentreStack server console (RDP or physical console are fine) with administrator rights. Once you click the **"Download"** button on the Partner Portal, the installer will appear either in downloads or as a link at the bottom of your browser. Click the executable file to begin the installation.
 
 You will see the welcome screen.
 
-.. image:: _static/image_s2_5_2.png
+.. image:: _static/image_s2_5_2_v3.png
```
Comment (anabelpgladinet) [inline install/source/chapter02.rst:121]: Remove old replaced images

File: install/source/chapter03.rst [hunk 5]
```diff
@@ -4,7 +4,7 @@
 
 After the machine reboots, the Default administration configuration page will be displayed.
 
-.. image:: _static/image_s3_1_1.png
+.. image:: _static/image_s3_1_1_v2.png
```
Comment (anabelpgladinet) [inline install/source/chapter03.rst]: Do you have an image without the loading?
Put this image after the next text: First, setup the cluster administrator account.

File: install/source/chapter07.rst [hunk 6]
```diff
@@ -2,46 +2,36 @@
 7 Setup Worker Node for SSL
```
Comment (anabelpgladinet) [inline install/source/chapter07.rst:2]: You didn't have to update Chapter 6, but noticed that at the beginning of the 6, in the Note, there is a breakline that seems incorrect:
Assuming that you have already setup your DNS to point to a DNS name connected to the CentreStack server and SSL certificate already

purchased for the DNS name.
If not, you can acquire a SSL certificate from your SSL vendor.

Should be one paragraph:
Assuming that you have already setup your DNS to point to a DNS name connected to the CentreStack server and SSL certificate already purchased for the DNS name. If not, you can acquire an SSL certificate from your SSL vendor.

File: install/source/chapter07.rst [hunk 7]
```diff
@@ -2,46 +2,36 @@
...
-Click on the "Cluster Server Farm" page to see the "Cluster Worker Nodes".
+After this, go to "Worker Nodes".
 
-.. image:: _static/image_s7_1_2.png
+.. image:: _static/image_s7_1_2_v2.png
 
 There is an "Advanced Setting" icon on the page.
 
-.. image:: _static/image_s7_1_3.png
+.. image:: _static/image_s7_1_3_v2.png
```
Comment (anabelpgladinet) [inline install/source/chapter07.rst]: enable the 2 options mentioned and retake the screenshot:
(Now, check “Always force SSL on Login” and “Always force SSL for Native Clients” boxes.)

File: install/source/chapter07.rst [hunk 8]
```diff
@@ -2,46 +2,36 @@
...
-.. warning::
-
-    In the current version, the Internal URL is hidden and it will be automatically set to a correct value so no need to manage the Internal URL any more.
-    
-    In previous version, this was where the Internal URL was set.
-    
-    The internal URL will be the internal IP address or private DNS name.
-
-    .. image:: _static/image037.jpg
+Note: The internal URL is the internal IP address or private DNS name.
```
Comment (anabelpgladinet) [inline install/source/chapter07.rst]: We included this note above, you can remove it here

File: install/source/chapter07.rst [hunk 9]
```diff
@@ -2,46 +2,36 @@
...
 
 The External URL will need to match the node's DNS name and the HTTP or HTTPS protocol.
 
-.. image:: _static/image_s7_1_5.png
+.. image:: _static/image_s7_1_5_v2.png
 
 The Node Name needs to match the hostname of the node.
 
-.. image:: _static/image036.jpg
+.. image:: _static/image_s7_1_6_v2.png
```
Comment (anabelpgladinet) [inline install/source/chapter07.rst:31]: Let's change the order here, after the: You will also modify the node property for the node.
put this part:
The Node Name needs to match the hostname of the node.
(and also move the respective screenshot)

then External URL
then Internal URL
so the order will be the same that in the screenshot

File: install/source/chapter07.rst [hunk 10]
```diff
@@ -2,46 +2,36 @@
...
 
 The external URL shall match the external URL for HTTPS. (If you do not have SSL certificate installed yet, this can be HTTP for now).
```
Comment (anabelpgladinet) [inline install/source/chapter07.rst]: Uppercase the first External, so it is the same as the field in the screenshot

File: install/source/chapter07.rst [hunk 11]
```diff
@@ -2,46 +2,36 @@
...
 .. warning::
 
   If you use the self-signed SSL certificate, the web portal is the only client that allows you to login after some SSL certificate warning. All other native clients such as Windows, mobile, and Mac clients will reject the connection.
 
-.. image:: _static/image_s7_1_4.png
+.. image:: _static/image_s7_1_4_v2.png
 
 You will also modify the node property for the node.
 
 The External URL will need to match the node's DNS name and the HTTP or HTTPS protocol.
```
Comment (anabelpgladinet) [inline install/source/chapter07.rst]: This seems wrong, change it to:
The Internal URL will need to match the internal IP address or node's private DNS name and the HTTP or HTTPS protocol.

File: install/source/chapter07.rst (hunk 9 above)
Comment (anabelpgladinet) [inline install/source/chapter07.rst:31]: Change the External URL, like in the original image, so users can see that External URL can be different and should be like the official URL for all users to access

File: install/source/chapter10.rst [hunk 12]
```diff
@@ -12,7 +12,7 @@ the web portal as the Master Administrator.
 
 After you login to the web portal, click to the "Cluster Branding" Pages.
 
-.. image:: _static/image_s10_1_1.png
+.. image:: _static/image_s10_1_1_v2.png
 
 Please reference the "Branding" section in the `Admin Guide`__ for the self-hosted CentreStack server.
```
Comment (anabelpgladinet) [inline install/source/chapter10.rst:17]: Point this Admin Guide link to the specific section for the branding in the Admin Guide:
https://centrestack.com/Library/AdminGuide/chapter03.html#cluster-branding

File: install/source/chapter10.rst [hunk 13]
```diff
@@ -12,7 +12,7 @@ the web portal as the Master Administrator.
```
Comment (anabelpgladinet) [inline install/source/chapter10.rst:12]: let's change the very first sentence to get rid of all those clients...
You can brand web portal, Windows client, Mac client, iOS client and Android client.

could be something like this?
You can brand web portal, Windows and Mac clients, iOS and Android Apps.

File: install/source/chapter11.rst [hunk 14]
```diff
@@ -10,8 +10,8 @@ __ CentreStack_Blog_
 Using the GUI Installer to Upgrade
```
Comment (anabelpgladinet) [inline install/source/chapter11.rst:10]: Check the first paragraph, seems the link in the CentreStack Blog got lost at some point, can you fix it?

**Review by anabelpgladinet — CHANGES_REQUESTED** [review]

File: install/source/chapter11.rst (hunk 14 above)
Comment (BryanGuo302) [reply install/source/chapter11.rst:10]: Ok, so I have identified where the problem is. It seems like the issue was there even in the master build, so I'm just going to try to fix it.

File: install/source/chapter07.rst (hunk 9 above)
Comment (anabelpgladinet) [reply install/source/chapter07.rst:31]: External URL is a web portal address, in the old image they use https://file.tsys.gladinet.com, you are using an email ABC@xyz.com, should be an URL, you can use the same old one or something like https://acme.example.com

**Pull request merged**